NuGet.config

```diff
@@ -2,6 +2,11 @@
 <configuration>
   <packageSources>
     <clear />
-    <!-- Restore sources should be defined in build/sources.props. -->
+    <add key="aspnetcore dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
+    <add key="aspnetcore tools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
+    <add key="myget dnc" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I ended up needing to put all of the restore sources that I care about in `<RestoreSources>` for various reasons. The nuget.config in my repo just has the minimal stuff that arcade needs

Reply:
I don't think we'll need the same thing for this repo. NuGet.config is preferrable, but if there are scenarios which require restore-config-in-MSBuild, I'm okay with `<RestoreSources>` too. Let's just try to avoid keeping two lists of feeds.